docs/modules/ROOT/pages/how-tos/worklows/git-submodules.adoc

```diff
@@ -0,0 +1,101 @@
+= Git Submodule guideline
```

Comment:
The workflow isn't about git submodules as much as it is about building upstream projects _with_ git submodules. Users may have git submodules in their repositories for other reasons ... would you expect these guidelines to still be relevant?

Your first sentence for motivating this page is
> This document outlines some basic guidelines for synchronizing changes between an upstream and downstream repository using Git submodules.

Reply:
I have updated the title, see https://github.com/konflux-ci/docs/pull/120/files#diff-8168286ae4fbd44cc1f9e5677aa3a5949ae97f5f391b1466cc253efe6176ccf8R1

Does that seem better?

Reply:
Yeah, the title seems better.